lib/internal/test_runner/test.js

```diff
@@ -76,15 +75,40 @@ const { testNamePatterns, testOnlyFlag } = parseCommandLine();
 let kResistStopPropagation;
 
 function stopTest(timeout, signal) {
+  const deferred = createDeferredPromise();
+
   if (timeout === kDefaultTimeout) {
-    return once(signal, 'abort');
+    signal.addEventListener('abort', deferred.resolve, { __proto__: null, once: true });
```

Comment:
please replace this with `events.addAbortListener`, does pretty much the same, and we can use `SymbolDispose` on it

Reply:
thanks, done